src/content/en/2020/markup.md

```diff
@@ -76,17 +76,15 @@ A page's document size refers to the amount of HTML bytes transferred over the n
 
 {# TODO(authors, analysts): Revisit the "largest document" stat and interpretation. #}
 * 1,110 documents are empty (0 bytes).
-* The average document size is 50.35 KB ([in most cases compressed](https://w3techs.com/technologies/details/ce-gzipcompression)).
-* The largest document by far weighs 64.16 _MB_, almost deserving its own analysis and chapter in the Web Almanac.
+* The average document size is 49.17 KB ([in most cases compressed](https://w3techs.com/technologies/details/ce-gzipcompression)).
+* The largest document by far weighs 62.65 _MB_, almost deserving its own analysis and chapter in the Web Almanac.
 
-{# TODO(analysts): Should 25,237 bytes be divided by 1000 or 1024 to convert to KB? 1000 seems to be used here but most chapters use 1024. Are the stats above also off? #}
```

Comment:
Note: I went through this section and re-calculated the values.

@Tiggerito, does this look right?

What I didn’t dive into now are the big documents. The section above seems to work, not sure what to add and what to do there that would benefit the rest of the section.

Reply:
LGTM. Ping @Tiggerito for a quick check if possible.